### PR TITLE
Refactor Photo component

### PR DESCRIPTION
**Pull request by natalieacevedo**
This component was returning the same parent component, so I conditionally rendered some elements and components according whether the photo is part of an album or not. #27

### CODE REVIEW

File: src/components/Photo.tsx [hunk 1]
```diff
@@ -48,11 +48,12 @@ function Photo({ photo, albumId }: PhotoParams) {
...
-  if (photo && albumId) {
+  if (photo) {
     const { id, prev, next } = photo
-
-    const realPhotoLink = `${baseUrl}/api/v1/photos/${id}/file`
-
+    const photoLink =
+      photo && albumId
+        ? `${baseUrl}/api/v1/photos/${id}/file`
+        : `${baseUrl}/api/v1/photo/${id}/file`
```
Comment (haplo) [inline src/components/Photo.tsx]: URL for the photo file is always the same, no need to check for `albumId`

```typescript
const photoLink = `${baseUrl}/api/v1/photos/${id}/file`
```

File: src/components/Photo.tsx [hunk 2]
```diff
@@ -76,26 +77,13 @@ function Photo({ photo, albumId }: PhotoParams) {
           </div>
         )}
         <Drawer photo={photo} />
-        <NavigationDrawer photo={photo} toggleDrawer={isOpen} albumId={albumId} />
+        {albumId && (
```
Comment (haplo) [inline src/components/Photo.tsx]: You always want `NavigationDrawer`, remove `albumId &&`.

**Review by haplo — CHANGES_REQUESTED** [review]

File: src/pages/PhotoInAlbumPage.tsx [hunk 3]
```diff
@@ -44,8 +44,10 @@ function PhotoInAlbumPage({ toggleHeader }: PhotoInAlbumProps) {
   if (isError) {
     return <div>{error ? error.message : "Unknown error"}</div>
   }
-
-  return <Photo photo={data} albumId={albumId} />
+  if (data) {
+    return <Photo photo={data} albumId={albumId} />
+  }
+  return <h1>No Photos here</h1>
```
Comment (haplo) [inline src/pages/PhotoInAlbumPage.tsx:50]: Let's remove `PhotoInAlbumPage` altogether, can combine with *Photo.tsx*.

**Review by haplo — APPROVED** [review]

File: src/pages/PhotoInAlbumPage.tsx (hunk 3 above)
Comment (natalieacevedo) [reply src/pages/PhotoInAlbumPage.tsx:50]: I thought exactly the  same, would do! :)

**Pull request merged**